### PR TITLE
[OneExplorer] Open ConfigEditPanel by clicking 

### PR DESCRIPTION
**Pull request by dayo09**
This commit enables users to open ConfigEditPanel by clicking
the .cfg file on OneExplorer view.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

### CODE REVIEW

**Comment by llFreetimell** [comment]
This PR may be conflict with #553 because of including same commit...?

**Comment by dayo09** [comment]
@llFreetimell That's true. I put it to have approvals concurrently.

**Comment by hyunsik-yoon** [comment]
Then please put `no merge` lable until #553 is merged so that reviewers do not need to be curious about the order.

**Comment by dayo09** [comment]
My mind was in a hurry that I thought today was the demo day.
I will split prs on the previous one.
PTAL this first.

File: src/OneExplorer.ts
Comment (hyunsik-yoon) [inline src/OneExplorer.ts:113]: For reviewers,
vscode comment mentioned that
```
         * The command that should be executed when the tree item is selected.
```

**Review by hyunsik-yoon — APPROVED** [review]

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**
